### DISCUSSION TITLE
Is there an easy way to check the best tool for a block?

### DISCUSSION THREAD

**Discussion by emortaldev** [Minestom/Minestom#914] [Help]
e.g. stone -> pickaxe, dirt -> shovel

**Comment by DevBlokJochem** [comment]
You are coding in Kotlin right?
You can create a when function that will return an itemstack. The only thing you have todo is adding the blocks you want.

Something like this:

fun getBestItem(block : Material) : Material {
return when(block) {

Material.STONE -> Material.DIAMOND_PICKAXE
Material.WOOD -> Material.DIAMOND_AXE

}
}

**Comment by DasLixou** [comment]
I’ll suggest to get a look at some Mods for that like Hwyla or other ones and Reverse engineer them
